tests/test_synced_collection.py

```diff
@@ -476,39 +499,42 @@ def test_reset(self, synced_list):
         with pytest.raises(ValueError):
             synced_list.reset(1)
 
+    @given(JSONDataStrategy)
     def test_insert(self, synced_list, testdata):
         synced_list.reset([1, 2])
         assert len(synced_list) == 2
         synced_list.insert(1, testdata)
         assert len(synced_list) == 3
         assert synced_list[1] == testdata
 
-    def test_reversed(self,  synced_list):
-        data = [1, 2, 3]
-        synced_list.reset([1, 2, 3])
-        assert len(synced_list) == 3
+    @given(st.lists(JSONDataStrategy, max_size=5))
+    def test_reversed(self,  synced_list, data):
```

Comment:
There's an extra space from the old code that can be removed.
```suggestion
    def test_reversed(self, synced_list, data):
```